### DISCUSSION TITLE
Hello. How does a blind man see world and makes better music than a person with eye ?

### DISCUSSION THREAD

**Discussion by ghost** [audacity/audacity#3819] [Ideas & Feature Requests]
### Your idea

Dear Audacity Team,

I have been keeping an eye on you from my school days. 1998. My first search result on audio editing software.

The tool is beautiful. I'm done suggesting new ideas, instead, I'll appreciate what I have found.

My best to you all, and your families,

Cheers.

### Problem to be solved

I know the core of this project is in terms of code,

I'm proposing that we change it to a dimensional, not 2d not 3d.

Let Audacity be how it is now, add technicalities behind to advanced users which will show 3d or any other dimension to visualise the audio that is input to the user.

I'm giving up in explaining these things. Mimic outside world inside the computer. Visualise the audio in 3d, physicist and their laws on audio hurts developers because they learn someone else's theories.

I know this is vague, I can only put this much into text.

### Prior art

Every other audio editing tools use mathematics to convert the input into an editable format.

### Additional context

My inspiration - How does a blind man see world and makes better music than a person with eye ?

**Comment by PatrikUgrin** [comment]
Sometimes memes write themselves
